.github/workflows/ci_gcc.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        version: [I, J, K, L] # [H, I, I_patch, J, K, L]
+        version: [L] # [H, I, I_patch, J, K, L]
```

Comment:
Have we decided for sure that we only care about version L for this? @Yanis002 I may want your opinion on this too.

Reply:
I'm not sure if I'm the best person but afaict it should be fine for HackerOoT, GC versions seems to use the L version

Reply:
afaik most changes between versions are bugfixes and a handful of feature additions that should always be a good thing to have (like mgu and `__printfunc`)
The only removed feature I know of offhand is reading `osClockRate` from the ROM header, but this is useless outside of the indy dev board anyway